recipes/libpano13/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "2.9.20" %}
```

Comment:
The source below points to an RC. Should the version be updated here?

Reply:
The project has been quite dormant. But if you wanted to be specific what is the conda way to add an rc to the version?

Reply:
:wink: Look at the output for:
```
conda search '*=*rc*' -c conda-forge
```

Reply:
I guess the solver will pick this because it is the only libpano version out there?

I really think it is an honest mistake that the developers forgot to release a final version.

Reply:
Yes, and conda claims to have support for pre-release versioning: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

Reply:
I guess my question is: 

Ok, for my application, I guess I'll give a constraint that says something like ">=2.9.20" to make sure I get the rc.